server_proxy/src/pxconfig.c

```diff
@@ -1556,8 +1556,10 @@ int ism_proxy_complexConfig(ism_json_parse_t * parseobj, int complex, int checko
                 }
             } else {
                 if (complex == 2)
+                {
                     rc = ISMRC_BadPropertyName;
                     ism_common_setErrorData(rc, "%s", ent->name);
+                }
```

Comment:
This changes the behaviour, I would guess you have corrected it but this code is awful to read so not sure, maybe Jon can be more sure

Reply:
Yeah - I think this change is good.